src/server.rs

```diff
@@ -166,6 +191,20 @@ impl ServerStats {
             }};
         }
 
+        macro_rules! set_duration_stat {
+            ($vec:ident, $dur:expr, $num:expr, $name:expr) => {{
+                let mut stat = CacheStatistic::new();
+                stat.set_name(String::from($name));
+                if $num > 0 {
+                    let duration = $dur / $num as u32;
+                    stat.set_str(format!("{}.{:03} s", duration.as_secs(), duration.subsec_nanos() / 1000_000));
+                } else {
+                    stat.set_str("0.000 s".to_owned());
```

Comment:
The `Duration` in `CacheStatistic` came to my mind when thinking about #12 in order to have a two column display with value and unit (like in the `ccache` output):

```

max cache size            5.0 GB
some duration             433 ms
other duration          0.324 s
```